packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts

```diff
@@ -20,19 +23,20 @@ import { v4 as uuid } from "uuid";
  */
 export class InsecureTinyliciousUrlResolver implements IUrlResolver {
     public async resolve(request: IRequest): Promise<IResolvedUrl> {
-        const url = request.url.replace("http://localhost:3000/","");
+        const url = request.url.replace(`http://localhost:${defaultTinyliciousPort}/`, "");
         const documentId = url.split("/")[0];
         const encodedDocId = encodeURIComponent(documentId);
         const documentRelativePath = url.slice(documentId.length);
 
-        const documentUrl = `fluid://localhost:3000/tinylicious/${encodedDocId}${documentRelativePath}`;
-        const deltaStorageUrl = `http://localhost:3000/deltas/tinylicious/${encodedDocId}`;
-        const storageUrl = `http://localhost:3000/repos/tinylicious`;
+        // eslint-disable-next-line max-len
+        const documentUrl = `fluid://localhost:${defaultTinyliciousPort}/tinylicious/${encodedDocId}${documentRelativePath}`;
```

Comment:
super-nit: consider this if it fixes the mex-len lint error.

```suggestion
        const documentUrl =
            `fluid://localhost:${defaultTinyliciousPort}/tinylicious/${encodedDocId}${documentRelativePath}`;
```